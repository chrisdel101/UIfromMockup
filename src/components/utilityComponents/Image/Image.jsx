import React from 'react'
import { Image } from 'react-bootstrap'

function renderType(props) {
  if (props.type === 'fluid') {
    return <Image src={props.src} alt={props.alt} fluid />
  } else {
    return <img src={props.src} alt={props.alt} />
  }
}
function MyImage(props) {
  return (
    <div className="image-container" id={props.id ? props.id : ''}>
      {renderType(props)}
    </div>
  )
}

export default MyImage
