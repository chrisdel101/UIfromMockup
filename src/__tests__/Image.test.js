import React from 'react'
import Image from '../components/utilityComponents/Image/Image.jsx'
import renderer from 'react-test-renderer'

test('create and match snapshot', () => {
  const component = renderer.create(<Image />)

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
