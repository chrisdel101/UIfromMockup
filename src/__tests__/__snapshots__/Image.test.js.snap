// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create and match snapshot 1`] = `
<div
  className="image-container"
  id=""
>
  <img />
</div>
`;
